docs/source/algorithms/IntegrateEllipsoids-v1.rst

```diff
@@ -187,15 +187,15 @@ For every point on the edge, the trajectory in reciprocal space is a straight li
 Calculate a point at a fixed momentum, say k=1.
 Q in the lab frame:
 
-:math:`\vec{E}=(-k*sin(\theta)*cos(\phi),-k*sin(\theta)*sin(\phi),k-k*cos(\phi))`
+:math:`\vec{E}=(-k \cdot \sin(\theta) * \cos(\phi), -k \cdot \sin(\theta) \cdot \sin(\phi), k - k \cdot \cos(\phi))`
```

Comment:
How come this uses two different multiplication symbols?

Reply:
Well spotted, this is an omission, all multiplication symbols should be created by `\cdot`.